Library/Categories/CoreData/NSManagedObject/NSManagedObject+MagicalDataImport.m

```diff
@@ -72,12 +72,14 @@ - (BOOL)MR_shouldImportData:(id)relatedObjectData forRelationshipNamed:(NSString
 
 #pragma mark - Setting Attributes and Relationships
 
-- (void)MR_setObject:(NSManagedObject *)relatedObject forRelationship:(NSRelationshipDescription *)relationshipInfo
-{
+- (void)MR_setObject:(NSManagedObject *)relatedObject forRelationship:(NSRelationshipDescription *)relationshipInfo{
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
It seems like it's valid: https://twitter.com/jckarter/status/695373983995138048

But if you could switch it to use clang instead, that would be consistent with the rest of the project. Thanks!